code/ui/.storybook/preview.tsx

```diff
@@ -164,17 +172,16 @@ export const decorators = [
       case 'stacked': {
         return (
           <Fragment>
-            <Symbols icons={['folder', 'component', 'document', 'bookmarkhollow']} />
```

Comment:
I have no idea why or what this is supposed to do. It renders SVG elements, but they don't appear in the canvas (their height is 0). I want to remove it but I don't know if it's magically used somewhere else?

@ghengeveld git history shows you've worked with this, albeit 2 years ago so it's fair if you don't remember.

Reply:
Turns out it is necessary for any stories that renders the sidebar with icons.
Added it back in via a graceful decorator for selected stories.